index.html

```diff
@@ -1005,55 +1005,84 @@ <h2>Continue Watching & Play Next</h2>
-          <a data-cite="schema-org/TVEpsiode#">TVEpisode</a> which has
-          a <a>media item action</a> with either a
+          If there is one <a>media item episode</a> embedded in the
+          <a>media feed item</a>, then return that
+        </li>
+        <li>
+          If there is more than one <a>media item episode</a> embedded in the
+          <a>media feed item</a>, then add them to <var>candidates</var>
+        </li>
+        <li>
+          Remove any <var>candidates</var> which do not have a
+          <a>media item action</a> with either a
           <a data-cite="schema-org/CompletedActionStatus#">
           https://schema.org/CompletedActionStatus</a> or
           <a data-cite="schema-org/ActiveActionStatus#">
-          https://schema.org/ActiveActionStatus</a>. If there is more than one
-          then we should use the <a data-cite="schema-org/episodeNumber#">
-          episodeNumber</a> and <a data-cite="schema-org/seasonNumber#">
-          seasonNumber</a> to find the latest one. If we cannot find a
-          <var>current episode</var> then return null
+          https://schema.org/ActiveActionStatus</a>
+        </li>
+        <li>
+          Sort the remaining <var>candidates</var> by
+          <a data-cite="schema-org/seasonNumber#">seasonNumber</a> and then
+          <a data-cite="schema-org/episodeNumber#">episodeNumber</a> in a
+          descending order
+        </li>
+        <li>
+          Return the first item in <var>candidates</var> or null if it is empty
         </li>
+      </ul>
+    </p>
+
+    <p>
+      A user agent might want to display the next episode in a TV series to play
+      next. In this case, the user agent should calculate the
+      <dfn>play next episode</dfn> using the following algorithm:
+    </p>
 
+    <p>
+      <ul>
+        <li>
+          If the <a>main TV episode</a> is null, then return null
+        </li>
         <li>
-          Let <var>next episode number</var> be the <var>current episode</var>
+          Let <var>next episode number</var> be the <var>main TV episode</var>
           number plus 1
         </li>
-
         <li>
-          Let <var>next season number</var> be the <var>current episode</var>
+          Let <var>next season number</var> be the <var>main TV episode</var>
           season number
         </li>
-
         <li>
           If the <var>next episode number</var> is greater than the
           <a data-cite="schema-org/numberOfEpisodes#">numberOfEpisodes</a>
           in the season; then set <var>next episode number</var> to 1 and
           increment the <var>next season number</var> by 1
         </li>
-
         <li>
-          Return the <a data-cite="schema-org/TVEpsiode#">TVEpisode</a>
-          which has the <a data-cite="schema-org/episodeNumber#">
-          episodeNumber</a> equal to <var>next epsiode number</var> and the
+          Return the <a>media item episode</a> which has the
```

Comment:
What if the calculated episode is not in the feed item? Should we still return a TV episode object containing the episode number even though we may not have the episode name, images, etc.?

Reply:
Also, after we calculate the play next candidate, we need to search through the media feed item's tv episodes for it. Should we filter by action status type as we did when finding the main episode?

Reply:
If there is no matching episode the play next candidate should be null. Added the action status type check from above.